crates/torii/core/src/sql_test.rs

```diff
@@ -60,6 +61,12 @@ where
     let data = engine.fetch_range(0, to, None).await.unwrap();
     engine.process_range(data).await.unwrap();
 
+    db.executor.send(QueryMessage {
+        statement: "".to_string(),
+        arguments: vec![],
+        query_type: QueryType::Execute,
+    });
```

Comment:
Let's have a function that ease the execution of the tx as in the future we may call it from different point in the code.
```rust
// Something like
db.executor.send(QueryMessage::execute())
```